Source/Public/Get-SegmentedPath.ps1

```diff
@@ -41,49 +41,89 @@ function Get-SegmentedPath {
         [PoshCode.Pansies.RgbColor]$ErrorBackgroundColor
     )
 
-    $buffer = @()
+    # Define reusable constants.
+    $directorySeparatorChar = [System.IO.Path]::DirectorySeparatorChar
+    $ellipses = data { [char]0x2026 }
+    $resolvedHome = (Resolve-Path -LiteralPath $Home).Path
```

Comment:
This is redundant. In what circumstances would `$Home` not already be a full path?
Also, when you want a string, why would you not use `Convert-Path`?